### PR TITLE
NS-323: Adds SIS advising notes data to s3

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/NS-324

### CODE REVIEW

File: nessie/jobs/scheduling.py [hunk 1]
```diff
@@ -189,11 +189,13 @@ def run_startup_jobs(_app):
...
     from nessie.jobs.create_rds_indexes import CreateRdsIndexes
+    from nessie.jobs.create_sis_advising_notes_schema import CreateSisAdvisingNotesSchema
     from nessie.jobs.create_student_schema import CreateStudentSchema
 
     if _app.config['JOB_SCHEDULING_ENABLED'] and os.environ.get('NESSIE_ENV') != 'test':
         _app.logger.info('Checking for required schemas...')
         CreateAscSchema().run()
         CreateMetadataSchema().run()
         CreateRdsIndexes().run()
+        CreateSisAdvisingNotesSchema().run()
```
Comment (raydavis) [inline nessie/jobs/scheduling.py]: It seems premature to put this into the scheduler. OK if we leave it "manual-only" for a while?

File: package-lock.json [hunk 2]
```diff
@@ -7782,7 +7782,7 @@
     "eslint-config-prettier": {
       "version": "3.1.0",
       "resolved": "https://registry.npmjs.org/eslint-config-prettier/-/eslint-config-prettier-3.1.0.tgz",
-      "integrity": "sha1-LCbSzc+joF8GQs1+bk7zMWzav6I=",
+      "integrity": "sha512-QYGfmzuc4q4J6XIhlp8vRKdI/fI0tQfQPy1dME3UOLprE+v4ssH/3W9LM2Q7h5qBcy5m0ehCrBDU2YF8q6OY8w==",
```
[the remark this answers is not in the document]
Comment (paulkerschen) [reply package-lock.json]: I think different MacOS (or Node, or npm versions) in use among the team perhaps have different preferences for package-lock.json:
https://github.com/npm/npm/issues/16938

I've also brushed against this issue locally:
https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818

[the remark this answers is not in the document]
Comment (lyttam) [reply package-lock.json]: I was wondering what those changes were about. I followed the [instructions](https://github.com/npm/npm/issues/16938#issuecomment-339863980) in that issue thread and it resulted in a ton of additional changes to my package-lock.json. I'll put it back the way it was and amend.

**Comment by raydavis** [comment]
Thanks, @lyttam -- I look forward to trying some queries out!

**Pull request merged**